Dockerfile

```diff
@@ -14,10 +14,17 @@ EXPOSE 8096
 VOLUME /config /media
 
 ARG FFMPEG_URL=https://www.johnvansickle.com/ffmpeg/old-releases/ffmpeg-4.0.3-64bit-static.tar.xz
-RUN apt update \
- && apt install -y xz-utils \
- && curl ${FFMPEG_URL} | tar Jxf - -C /usr/bin --wildcards --strip-components=1 ffmpeg*/ffmpeg ffmpeg*/ffprobe \
- && apt remove -y xz-utils \
- && apt install -y libfontconfig1  # needed for Skia
+RUN DEBIAN_FRONTEND=noninteractive \
+ apt-get update && \
+ apt-get upgrade -y && \
+ apt-get install --no-install-recommends --no-install-suggests -y \
```

Comment:
I disagree with the separation. It serves no real purpose to do so. With the slight indentation, its also more apparent what is installed rather than having it in even more places in the dockerfile.

As for the comment, I originally planned to keep it but it caused syntax BS. Think it had to do with the && switch. Regardless, if you want it back I'll find a way to make it happen.